src/components/PatientLookup.tsx

```diff
@@ -86,44 +88,41 @@ const PatientLookup = (props: PatientLookupProps) => {
     </DropdownItem>,
   ];
 
-  const navigateToCreateAnalysis = async () => {
-    await newLookup();
-    history.push("/createAnalysis");
-  }
-
-  const submitButton = props.isOnDashboard ? (
-    <RightArrowButton click={navigateToCreateAnalysis}>Continue</RightArrowButton>
+  const submitButton = isOnDashboard ? (
+    <RightArrowButton click={newLookup}>Continue</RightArrowButton>
```

Comment:
Instead of using the `click` event, what is your opinion about adding `type?: "button" | "submit" | "reset" | undefined;` as a new prop for `RightArrowButtonProps`? I believe that would provide more the event more context as well as it makes more sense given that the submit button is inside a form. Let me know what is your opinion about this and we can discuss further.